cwl/workflows/parts/32_clusters/32_mmseq.yml

```diff
@@ -0,0 +1,17 @@
+prokka_many:
+  -
```

Comment:
I think this is not valid yml

Reply:
no, cwltool worked with test **tests/cwltest/utils/flatten_array.yml**